style.css

```diff
@@ -0,0 +1,134 @@
+
+nav {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.logo {
+  margin-right: auto;
+  align-items: center;
+  color: #6070ff;
+  text-align: center;
+  margin-left: 120px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+}
+
+nav li {
+  list-style: none;
+  display: inline-block;
+  padding: 0 10px;
+  color: #344563;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  margin-left: 150px;
+}
```

Comment:
- well done fixing the font family for the website 👏😎 however there are some issues you still need to work on, so please consider the following:

- [ ] the class `header` style causing the shrinking in size of the website since the property margin-left is set with a fixed value, Kindly fix the margin-left so the website will be responsive on different mobile screen sizes 💪

### Your Design 
![image](https://user-images.githubusercontent.com/68030297/219726804-17dfcf1a-c72f-429c-a8e2-d2193add20b1.png)

### Figma design
![image](https://user-images.githubusercontent.com/68030297/219728734-970665ce-09e2-4336-a1db-b2afc52b538e.png)